util/guard-one/guard-one.repl

```diff
@@ -0,0 +1,172 @@
+;; gas-guards.repl
+(env-data
+ { 'util-ns-users: ["util-ns-user"]
+ , 'util-ns-admin: ["util-ns-admin"]
+ })
+
+(env-keys ["util-ns-user", "util-ns-admin"])
+
+;;Load contracts
+(begin-tx)
+(load "../util/util-ns.pact")
+(load "../guards/guards.pact")
+(load "guard-one.pact")
+(commit-tx)
+
+(begin-tx)
+(use util.guard-one)
+
+; set chain gas price to 0.0005
+(env-chain-data {
+  "gas-price": 0.0005,
+  "gas-limit": 500
+})
+
+;;Test max-gas
+
+(expect
+  (format "chain-gas {} is smaller than max-gas {}"
+    [(chain-gas), 0.5]) true
```

Comment:
Not a big deal but `true` being on the same line here is confusing to read -- would suggest instead 
```
(expect
  (format "chain-gas {} is smaller than max-gas {}"
    [(chain-gas), 0.5])
  true
  (enforce-guard (max-gas 0.5)))
```

up to you. if you make the change however, make it to all the tests below that are like this.

Reply:
Applied the changes